pkg/services/apiserver/builder/openapi.go

```diff
@@ -17,6 +17,9 @@ func GetOpenAPIDefinitions(builders []APIGroupBuilder) common.GetOpenAPIDefiniti
 	return func(ref common.ReferenceCallback) map[string]common.OpenAPIDefinition {
 		defs := v0alpha1.GetOpenAPIDefinitions(ref) // common grafana apis
 		maps.Copy(defs, data.GetOpenAPIDefinitions(ref))
+		maps.Copy(defs, map[string]common.OpenAPIDefinition{ // TODO -- move to SDK definition above
```

Comment:
This can be removed when https://github.com/grafana/grafana-plugin-sdk-go/pull/1062 is merged